package.json

```diff
@@ -1,11 +1,43 @@
 {
-  "name": "todo",
+  "name": "todo-js",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "webpack serve --config webpack.dev.js --open",
```

Comment:
## Optional Change 
- [ ] I have not been able to start up the project to run with this command. Was the implementation done right?